### PR TITLE
Job application customizing code 

### PR DESCRIPTION
**Pull request by MehdiElrafi**

### CODE REVIEW

**Comment by MehdiElrafi** [comment]
Before: 
Screenshot_2023-07-03_12-46-27
After:
Screenshot_2023-07-03_12-35-13

**Comment by MehdiElrafi** [comment]
Before : 
Screenshot_2023-07-03_12-46-27
After : 
Screenshot_2023-07-03_13-47-45

File: app/models/job_application.rb [hunk 1]
```diff
@@ -48,20 +48,54 @@ class JobApplication < ApplicationRecord
...
+    when "advanced_interview"
+      I18n.t("attributes.job_application.aasm_states.advanced_to_interview", by: "")
+    when "completed_interview"
+      I18n.t("attributes.job_application.aasm_states.qualified")
+    when "disqualified"
+      I18n.t("attributes.job_application.aasm_states.disqualified")
+    end
+  end
+
+  def job_present(job_id, job_application_id, next_state = "")
```
Comment (mohamedrez) [inline app/models/job_application.rb:82]: this naming is not meaninful, the function name should describe better the code that it wraps

File: app/views/job_applications/_right_side.html.erb [hunk 2]
```diff
@@ -33,13 +33,13 @@
                     <%
                       case aasm_log.to_state
```
Comment (mohamedrez) [inline app/views/job_applications/_right_side.html.erb:34]: no case inside the view.

**Review by mohamedrez — CHANGES_REQUESTED** [review]

**Review by mohamedrez — APPROVED** [review]

**Pull request merged**